endorser/src/index.ts

```diff
@@ -0,0 +1,13 @@
+import logger from './logger';
+import app from './app';
+
+const port = app.get('port');
+const server = app.listen(port);
+
+process.on('unhandledRejection', (reason, p) =>
```

Comment:
We will want to incorporate graceful SIGTERM handling but that can be filed as a separate enhancement